pages/cart.html

```diff
@@ -0,0 +1,177 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cruxui-lib.netlify.app//Components/main.css"
+    />
+    <link rel="stylesheet" href="../style.css" />
+    <script src="https://code.iconify.design/2/2.1.2/iconify.min.js"></script>
+    <title>CALYX-ECOMM</title>
+  </head>
+  <body>
+    <div class="rootContainer">
+      <!-- NavBar -->
+      <div class="navbar bg-primary p-2" style="z-index: 100">
+        <input id="menu-icon" type="checkbox" />
+        <label class="menu-label" for="menu-icon"></label>
+        <div class="nav-brand">
+          <div class="h3 text-bold color-primary box-3d p-1-x bg-secondary">
+            CALYX Mart
+          </div>
+        </div>
+        <div class="nav-content">
+          <!-- <h3>Navbar</h3>
+                  <p>Text</p>
+                  <div class="submenu">
+                    <input id="submenu-input" type="checkbox" />
+                    <label for="submenu-input" class="submenu-text">
+                      Submenu
+                    </label>
+                    <ul>
+                      <li>Option 1</li>
+                      <li>Option 12</li>
+                      <li>Option 123</li>
+                    </ul>
+                  </div> -->
+        </div>
+        <div class="nav-end nav-style">
+          <button class="button button-secondary-light">
+            <a>Home</a>
+          </button>
+          <button class="button button-secondary-light">
+            <a>Product Listing Page</a>
+          </button>
+          <button class="button button-secondary-light">
+            <a>Cart Management</a>
+          </button>
+          <button class="button button-secondary-light">
+            <a>Wishlist Management</a>
+          </button>
+          <button class="button button-secondary-light">
+            <a>Sign-up Page</a>
+          </button>
+          <button class="button button-secondary-light">
+            <a>Login Page</a>
+          </button>
+        </div>
+      </div>
+      <!-- Cart -->
+      <div class="grid-col gap-2 p-4">
+        <div class="col-4 col-4-md col-6-sm">
+          <div class="card card-horizontal full-width whishlist-card p-4-y">
+            <div class="card-img">
+              <img
+                src="../assets/Photo by Brian Hall on Unsplash.jpg"
```

Comment:
Avoid putting spaces in names of Files/Folders
you can use underscore like this --> brian_hall.jpg

Reply:
will change these